internal/http/services/owncloud/ocdav/report.go

```diff
@@ -59,9 +74,67 @@ func (s *svc) doSearchFiles(w http.ResponseWriter, r *http.Request, sf *reportSe
 	w.WriteHeader(http.StatusNotImplemented)
 }
 
+func (s *svc) doFilterFiles(w http.ResponseWriter, r *http.Request, ff *reportFilterFiles, namespace string) {
+	ctx := r.Context()
+	log := appctx.GetLogger(ctx)
+
+	if ff.Rules.Favorite {
+		// List the users favorite resources.
+		currentUser := ctxpkg.ContextMustGetUser(ctx)
+		favorites, err := s.favoritesManager.ListFavorites(ctx, currentUser.Id)
```

Comment:
couldn't we reuse the user in the context on `ListFavorites` instead of also use the user id as argument? Seems redundant, and it is the only place `currentUser` is used within this scope.

Reply:
I wanted to make the API a bit more explicit concerning the user. Implicitly passing the user around in the context doesn't feel right. But I'm open to change this.

Reply:
We do assume the user to be passed around like that in the other interfaces, but it's not an absolute necessity